packages/util/package.json

```
@@ -0,0 +1,58 @@
+{
+	"name": "@discordjs/util",
+	"version": "0.1.0",
+	"description": "Utilities shared across Discord.js packages",
+	"private": true,
+	"scripts": {
+		"build": "tsup",
+		"lint": "prettier --check . && TIMING=1 eslint src --ext mjs,js,ts",
+		"format": "prettier --write . && TIMING=1 eslint src --ext mjs,js,ts --fix",
+		"fmt": "yarn format"
+	},
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"types": "./dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
+	},
+	"directories": {
+		"lib": "src"
+	},
+	"files": [
+		"dist"
+	],
+	"contributors": [
+		"Crawl <icrawltogo@gmail.com>"
+	],
+	"license": "Apache-2.0",
+	"keywords": [
+		"api",
+		"bot",
+		"client",
+		"node",
+		"discordjs"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/discordjs/discord.js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/discordjs/discord.js/issues"
+	},
+	"homepage": "https://discord.js.org",
+	"devDependencies": {
+		"@types/node": "^16.11.56",
+		"@vitest/coverage-c8": "^0.22.1",
+		"eslint": "^8.23.0",
+		"eslint-config-neon": "^0.1.31",
+		"prettier": "^2.7.1",
+		"tsup": "^6.2.3",
+		"typescript": "^4.8.2",
+		"vitest": "^0.22.1"
+	},
+	"engines": {
+		"node": ">=16.9.0"
+	}
```

Missing the `publicConfig`, `yarn publish` would error without this.
```suggestion
	},
	"publishConfig": {
		"access": "public"
	}
```